sys/stat_openbsd.go

```diff
@@ -0,0 +1,45 @@
+// +build openbsd
```

Comment:
Just posting here because at a glance this file looked the same as the `_unix.go` variant, but it's the name of the properties not having the `spec` suffix;

```patch
--- a/sys/stat_unix.go
+++ b/sys/stat_bsd.go
@@ -1,4 +1,4 @@
-// +build linux solaris
+// +build darwin freebsd netbsd

 /*
    Copyright The containerd Authors.
@@ -23,22 +23,22 @@ import (
        "time"
 )

-// StatAtime returns the Atim
+// StatAtime returns the access time from a stat struct
 func StatAtime(st *syscall.Stat_t) syscall.Timespec {
-       return st.Atim
+       return st.Atimespec
 }

-// StatCtime returns the Ctim
+// StatCtime returns the created time from a stat struct
 func StatCtime(st *syscall.Stat_t) syscall.Timespec {
-       return st.Ctim
+       return st.Ctimespec
 }

-// StatMtime returns the Mtim
+// StatMtime returns the modified time from a stat struct
 func StatMtime(st *syscall.Stat_t) syscall.Timespec {
-       return st.Mtim
+       return st.Mtimespec
 }

-// StatATimeAsTime returns st.Atim as a time.Time
+// StatATimeAsTime returns the access time as a time.Time
 func StatATimeAsTime(st *syscall.Stat_t) time.Time {
-       return time.Unix(int64(st.Atim.Sec), int64(st.Atim.Nsec)) // nolint: unconvert
+       return time.Unix(int64(st.Atimespec.Sec), int64(st.Atimespec.Nsec)) // nolint: unconvert
 }
```